### PR TITLE
MODIFIED: HTML Basic files

### PR DESCRIPTION
**Pull request by Jenifar-fathima**

### CODE REVIEW

File: JenifarFathimaH/01 - HTML/01 - Basics/03 - Links & Images/Links.html [hunk 1]
```diff
@@ -0,0 +1,87 @@
+﻿<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>HTML Links</title>
+    <nav >
```
Comment (msindia-learning) [inline JenifarFathimaH/01 - HTML/01 - Basics/03 - Links & Images/Links.html]: nav should be placed under body tag

File: JenifarFathimaH/01 - HTML/01 - Basics/03 - Links & Images/Links.html [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+        <a class="navbar" href="#home" target="_self">Home</a>
+        <a class="navbar"href="#about" target="_self">About</a>
+        <a class="navbar" href="#contact" target="_self">Contact</a>
+    </nav>
+</head>
+<style>
```
Comment (msindia-learning) [inline JenifarFathimaH/01 - HTML/01 - Basics/03 - Links & Images/Links.html]: style should be placed under head tag

File: JenifarFathimaH/01 - HTML/01 - Basics/03 - Links & Images/Links.html [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+    </section>
+    
+    <section id="about">
+        <h2>About me</h2>
+        <p>I’m a final-year Computer Science and Engineering student<br /> at Nadar Saraswathi College of Engineering and Technology,<br />
+        passionate about web designing and software development. <br />Eager to apply my skills and knowledge to enhance my capabilities<br />
+        in the dynamic tech industry.</p>
+    </section>
+</body>
+<footer>
```
Comment (msindia-learning) [inline JenifarFathimaH/01 - HTML/01 - Basics/03 - Links & Images/Links.html]: footer should be placed under body tag

File: JenifarFathimaH/01 - HTML/01 - Basics/03 - Links & Images/Links.html
Comment (msindia-learning) [inline JenifarFathimaH/01 - HTML/01 - Basics/03 - Links & Images/Links.html:1]: also use button with link

**Review by msindia-learning — COMMENTED** [review]
Please have a look

File: JenifarFathimaH/01 - HTML/01 - Basics/03 - Links & Images/Links.html (hunk 1 above)
Comment (Jenifar-fathima) [reply JenifarFathimaH/01 - HTML/01 - Basics/03 - Links & Images/Links.html]: Done

File: JenifarFathimaH/01 - HTML/01 - Basics/03 - Links & Images/Links.html (hunk 2 above)
Comment (Jenifar-fathima) [reply JenifarFathimaH/01 - HTML/01 - Basics/03 - Links & Images/Links.html]: Done

File: JenifarFathimaH/01 - HTML/01 - Basics/03 - Links & Images/Links.html (hunk 3 above)
Comment (Jenifar-fathima) [reply JenifarFathimaH/01 - HTML/01 - Basics/03 - Links & Images/Links.html]: Done

File: JenifarFathimaH/01 - HTML/01 - Basics/03 - Links & Images/Links.html
Comment (Jenifar-fathima) [reply JenifarFathimaH/01 - HTML/01 - Basics/03 - Links & Images/Links.html:1]: Done

**Review by msindia-learning — APPROVED** [review]

**Pull request merged**